keyboards/handwired/jscotto/scotto40/keymaps/default/rules.mk

```diff
@@ -0,0 +1,2 @@
+LTO_ENABLE = yes
```

Comment:
this should probably be moved to the keyboard's rules file

Reply:
Good point... done.